.github/workflows/ci-python-build-3.9-client.yaml

```diff
@@ -66,3 +66,13 @@ jobs:
         run: |
           echo "Build Python3.9 client library"
           pulsar-client-cpp/docker-build-python3.9.sh
+
+      - name: Local build python client
+        if: ${{ steps.check_changes.outputs.docs_only != 'true' }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+        working-directory: pulsar-client-cpp/python
+        run: |
+          echo "Local build python client library"
+          pip install .
```

Comment:
This section is incorrect and produces:

```
a step cannot have both the `uses` and `run` keys
```

See https://github.com/apache/pulsar/actions/runs/2978090895.

I guess it will cause all PRs to fail. Please take a look @nodece @RobertIndie.

Weird why it's not failed on PR CI workflows.

Reply:
Filed an issue in https://github.com/apache/pulsar/issues/17430.

Reply:
fix https://github.com/apache/pulsar/pull/17431

Reply:
Good catch!

Reply:
I just wonder why it's not verified within this patch :)

Reply:
GitHub Actions didn't run this workflow.

Reply:
Yes. But why? The trigger section writes:

```yml
on:
  pull_request:
    branches:
      - master
    paths:
      - '.github/workflows/**'
      - 'pulsar-client-cpp/**'
```